### PR TITLE
created sign up page & account verification page

### PR DESCRIPTION
**Pull request by MugiranezaJ**
This includes 

- Signup page
   - Skeleton
   - Connection to the backend
   - Form validation
   - responsive
- Account verification page
   - Connection to the backend
   - appropriate success & error messages
   - responsive

### CODE REVIEW

**Comment by salviosage** [comment]
Please Rebase

File: src/components/signup/secondStep.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+              onChange={(e) => {
+                const profile_picture = e.target.files[0]
+                const formData = new FormData()
+                formData.append('upload_preset', 'l9dhzfdi')
+                formData.append('file', profile_picture)
+                setLoading(true) 
+                setPhoto('')               
+                axios.post(' https://api.cloudinary.com/v1_1/mjackson/image/upload', formData)
+                .then(res => {setFieldValue("profile_picture", res.data.secure_url); setLoading(false); setPhoto(res.data.secure_url)})
+                .catch(err => {console.log(err); setLoading(false)})
```
Comment (salviosage) [inline src/components/signup/secondStep.js]: remove this log

File: src/components/signup/socialButton.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                    
+                    <FacebookLoginWithButton
+                        appId="1206715649505081"
+                        textButton='Signup with facebook'
+                        size= 'small'
+                        fields="name,email,picture"
+                        icon="fa-facebook"/>
+                    <p>Or</p>
+                    <GoogleLoginButton
+                        clientId="658977310896-knrl3gka66fldh83dao2rhgbblmd4un9.apps.googleusercontent.com"
```
Comment (salviosage) [inline src/components/signup/socialButton.js]: can you add this in .env

File: src/components/signup/verifyAccount.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        fontSize: '5vh',
+        padding: '20px',
+        marginBottom: '10px',
+        borderBottom: '2px black solid',
+    }
+}))
+const verify = (token, verified, setVerified ,setError) => {
+    Axios.patch(`https://barefoot-nomad-app-v1.herokuapp.com/api/v1/user/verification?token=` + token)
+    .then(res => {
+      console.log(JSON.stringify("Verification: " + verified))
```
Comment (salviosage) [inline src/components/signup/verifyAccount.js]: remove this log

File: src/components/signup/verifyAccount.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    })
+}
+const verifyAccount = () => {
+    const classes = useStyle()
+    const [verified, setVerified] = useState(false);
+    const [error, setError] = useState('');
+    const parsed = queryString.parse(location.search);
+    const [checked, setChecked] = useState(false)
+    const token = parsed.token
+    console.log(token)
```
Comment (salviosage) [inline src/components/signup/verifyAccount.js]: remove this log

File: src/components/signup/verifyAccount.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                </>) : (
+                    (!error ? (
+                        <Box>
+                            <Skeleton variant="text" width={300} height={40}/>
+                            <Skeleton variant="text" width={300} height={40}/>
+                        </Box>
+                    ) : (
+                        <Box className={classes.tokenMissing}>
+                            { JSON.stringify(error).replace(/['"]+/g, '')}
+                            {console.log('Verification: ' + error)}
```
Comment (salviosage) [inline src/components/signup/verifyAccount.js]: remove this log

File: __tests__/components/__snapshots__/accountVerification.test.js.snap [hunk 6]
```diff
@@ -0,0 +1,140 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (salviosage) [inline __tests__/components/__snapshots__/accountVerification.test.js.snap:1]: remove comments please

**Review by salviosage — COMMENTED** [review]
I left some comments , please do attend them . also do include bernards suggested jest configuration

File: __tests__/components/__snapshots__/accountVerification.test.js.snap (hunk 6 above)
Comment (MugiranezaJ) [reply __tests__/components/__snapshots__/accountVerification.test.js.snap:1]: This one is auto generated by jest. It is generated again whenever I run tests again

**Review by salviosage — APPROVED** [review]
LGTM

**Pull request merged**